ngraph/python/tox.ini

```diff
@@ -21,7 +21,7 @@ commands=
   {envbindir}/python setup.py bdist_wheel
   {envbindir}/pip install --no-index --pre --find-links=dist/ ngraph-core
   flake8 {posargs:src/ setup.py}
-  flake8 --ignore=D100,D101,D102,D103,D104,D105,D107,W503 tests/  # ignore lack of docs in tests
+  flake8 --ignore=D100,D101,D102,D103,D104,D105,D107,D401,W503 tests/  # ignore lack of docs in tests
```

Comment:
( ͡° ͜ʖ ͡°)

Reply:
Which docstring is causing an issue here?

Reply:
All docstrings using verbs that a tool doesn't like. For example "returns" instead of "return". I understand the checks for missing docs, incorrect formatting, complexity of the implemented code etc, but I don't want a tool telling me which verbs I should use.